services.go

```diff
@@ -513,3 +513,90 @@ func (s *ServicesService) DeleteJenkinsCIService(pid interface{}, options ...Opt
 
 	return s.client.Do(req, nil)
 }
+
+// MicrosoftTeamsService represents Microsoft Teams service settings.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ce/api/services.html#microsoft-teams
+type MicrosoftTeamsService struct {
+	Service
+	Properties *MicrosoftTeamsServiceProperties `json:"properties"`
+}
+
+// MicrosoftTeamsServiceProperties represents Microsoft Teams specific properties.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ce/api/services.html#microsoft-teams
+type MicrosoftTeamsServiceProperties struct {
+	WebHook                   *string `webhook:"webhook,omitempty" json:"webhook,omitempty"`
```

Comment:
It looks like you didn’t `gofmt` your code. Could you please do that to make sure its formatted correctly?

Reply:
You should not use a pointer here and the tag is not correct either... Please make sure they are inline with the other examples in this file.